DcCore/DcCore/DC/events.swift

```diff
@@ -214,6 +214,9 @@ public class DcEventHandler {
                 return
             }
             dcContext.logger?.info("network: DC_EVENT_CONNECTIVITY_CHANGED: \(dcContext.getConnectivity())")
+            if let sem = dcAccounts.fetchSemaphore, dcContext.getConnectivity() == DC_CONNECTIVITY_CONNECTED {
+                sem.signal()
+            }
```

Comment:
this seems to be easier and faster than creating an event handler etc. that may have issues on its own wrt lifetime etc.